doc/classes/EditorInspector.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="3.2">
 	<brief_description>
+	The Tab located by default in the right hand side of the editor. It is used to edit properties of the node selected.
```

Comment:
```suggestion
		The tab located by default in the right hand side of the editor. It is used to edit properties of the node selected.
```

Reply:
Thanks for pointing this out, just fixed it.